Ebraheem-Mahmood/style.css

```diff
@@ -0,0 +1,149 @@
+body{
+	align-items: center;
+	font-family: Segoe UI,Frutiger,Frutiger Linotype,Dejavu Sans,Helvetica Neue,Arial,sans-serif; 
+	background-color: #f9f2f2;
+}
+
+
+
+header{
+	position: fixed;
+	top:0;
+	right: 0;
+	left: 0;
+	width: 100%;
+	height: auto;
+	border-bottom: #fff0ff 1px solid;
+	background-color: white;
+	
+	
+}
+
+#logo-side{
+	margin-left: 20%;
+	margin-top:2px ;
+	display: inline-block;
+
+}
+
+.logo-img{
+	width: 50px;
+	height: 50px;
+	display: inline-block;
+}
+
+.btn-side{
+	margin-right: 20%;
+	margin-top:20px ;
+	display: inline-block;
+	float: right;
+	text-transform: uppercase;
+	font-size: 12px;
+}
+
+.btn-blue{
+	box-sizing: border-box;
+	padding: 5px 15px;
+	text-decoration: none;
+	border: 1px #0277bd solid;
+	color:white;
+	border-radius: 5px;
+	background-color: #039be5;
+	margin-left: 10px;
+	
+}
+.btn-white{
+box-sizing: border-box;
+	padding: 5px 15px;
+	text-decoration: none;
+	border: 1px #039be5 solid;
+	color:#5077bd;
+	border-radius: 5px;
+	background-color: #ffffff;
+}
```

Comment:
Great job on achieving the layouts of your buttons. 👍

But if you look at these two classes, you can see that they both share a number of properties such as (box-sizing, border-radius, padding, text-decoration).

Even though you have successfully achieved the look of the buttons, a better way to maintain the reusability of your code would be to make a class only for making the shape of the buttons and then two other classes for coloring them properly.
Like this:

.btn{
	box-sizing: border-box;
	padding: 5px 15px;
	text-decoration: none;
        border-radius:5px;
}

.btn-blue{
        border: 1px #0277bd solid;
	color:white;
	background-color: #039be5;
	margin-left: 10px;
}

.btn-white{
        border: 1px #039be5 solid;
	color:#5077bd;
	background-color: #ffffff;
}

By doing this, you give a hierarchy to your styling by giving all buttons the "btn" class so that they get the shape of the button then give the specific styling by giving them either "btn-white" class if you want it to be a white button or a "btn-blue" class if you want the button to be blue.